src/v/kafka/server/handlers/produce.cc

```diff
@@ -192,7 +192,8 @@ static ss::future<produce_response::partition> produce_topic_partition(
 
     auto num_records = batch.record_count();
     auto reader = reader_from_lcore_batch(std::move(batch));
-    return octx.rctx.partition_manager().invoke_on(
+    auto start = std::chrono::steady_clock::now();
```

Comment:
should we use ss::clock<>::now() maybe they have also a steady clock source. no big deal if not.

Reply:
I think that lowres_clock is the efficient steady clock in seastar